torch_np/_ufuncs.py

```diff
@@ -122,15 +123,18 @@ def divmod(
     signature=None,
     extobj=None,
 ):
-    num_outs = sum(x is None for x in [out1, out2])
-    if sum_outs == 1:
+    # make sure we either have no out arrays at all, or there is either
+    # out1, out2, or out=tuple, but not both
+    num_outs = sum(x is not None for x in [out1, out2])
+    if num_outs == 1:
         raise ValueError("both out1 and out2 need to be provided")
-    if sum_outs != 0 and out != (None, None):
-        raise ValueError("Either provide out1 and out2, or out.")
-    if out is not None:
+    else:
         out1, out2 = out
-    if out1.shape != out2.shape or out1.dtype != out2.dtype:
-        raise ValueError("out1, out2 must be compatible")
+        if num_outs == 2:
+            if out1 is not None or out2 is not None:
+                raise TypeError(
+                    "cannot specify 'out' as both a positional and keyword argument"
+                )
```

Comment:
I don't thin this logic is correct. If `num_outs == 2` and `out = (None, None)`, here we are setting setting `out1` and `out2` to `None`.

Reply:
Indeed, thanks for spotting this! Fixed in https://github.com/Quansight-Labs/numpy_pytorch_interop/pull/109/commits/a855a3710ba67baf2a68342f4b1d4ef8c3398916. The relevant test needs to stay xfailed for a while, since we have hardcoded the name "out" for postprocessing. 

The current state is that `out=tuple` works, and there is an xfailed test.